### ISSUE TITLE
how to install man pages? (built from source from github, borg v1.2.0 works)

### ISSUE THREAD

**Issue by SpiritInAShell** [borgbackup/borg#6661]
<!--
Thank you for reporting an issue.

*IMPORTANT* -  *before* creating a new issue please look around:
 - Borgbackup documentation: http://borgbackup.readthedocs.io/en/stable/index.html
 - FAQ: https://borgbackup.readthedocs.io/en/stable/faq.html
 and
 - open issues in Github tracker: https://github.com/borgbackup/borg/issues
  
If you cannot find a similar problem, then create a new issue.

Please fill in as much of the template as possible.
-->

## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Question

## System information. For client/server mode post info for both machines.

#### Your borg version (borg -V).
borg 1.2.0
#### Operating system (distribution) and version.
Linux Mint 20
#### Can you reproduce the problem? If so, describe how. If not, describe troubleshooting steps you took before opening the issue.
```
# all the stuff from https://borgbackup.readthedocs.io/en/1.2-maint/installation.html#git-installation
> pip install -e .[pyfuse3]
> borg -V
borg 1.2.0
> man borg                                                                                                                                                                                                                            No manual entry for borg
```
I am sure this is only a simple command not obvious to me...
edit: thinking about the problem, this is because I did not do a system-wide install (as root) but using the virtual environment???

So I copied the man files in the github tree (like `docs/man/borg.1` to `$HOME/.local/share/man` which is listed in `manpath` output), still no success... continue reading...

**Comment by SpiritInAShell** [comment]
Solution I found:

- copy files from `docs/man/borg*.1` to `$HOME/.local/share/man/man1/borg*.1`
- make sure the path `$HOME/.local/share/man` is listed in the output of `manpath`
- run `mandb` as local user

**Comment by ThomasWaldmann** [comment]
AFAIK we do not have code (yet?) that helps with man page installation.

**Comment by SpiritInAShell** [comment]
> AFAIK we do not have code (yet?) that helps with man page installation.

I would suggest a note (1-liner) in the page of installation process that man pages won't be installed by any script or automatism.

I am used to install packages from package managers not only for comfort but because I trust others to be more experienced in creating a stable package. This time I needed something from 1.2.0 not available in 1.1.x and went for multiple paths suggested in install processes to always miss the man pages.
